SPECS/conda/conda.spec

```diff
@@ -0,0 +1,449 @@
+%{!?_with_bootstrap: %global bootstrap 0}
+
+Summary:        Cross-platform, Python-agnostic binary package manager
+Name:           conda
+Version:        4.10.1
+Release:        2%{?dist}
+License:        BSD and ASL 2.0 and LGPLv2+ and MIT
+# The conda code is BSD
+# progressbar is LGPLv2+
+# six is MIT/X11
+# adapters/ftp.py is ASL 2.0
+URL:            http://conda.pydata.org/docs/
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://github.com/conda/conda/archive/%{version}/%{name}-%{version}.tar.gz
+# bash completion script moved to a separate project
+Source1:        https://raw.githubusercontent.com/tartansandal/conda-bash-completion/1.5/conda
+Patch0:         conda_sys_prefix.patch
+Patch1:         conda_gateways_disk_create.patch
+Patch2:         setup.patch
+# Use system cpuinfo
+Patch3:         conda-cpuinfo.patch
+
+Patch10001:     0001-Fix-toolz-imports.patch
+Patch10003:     0003-Drop-fs-path-encoding-manipulation-under-python2.patch
+Patch10004:     0004-Do-not-try-to-run-usr-bin-python.patch
+Patch10005:     0005-Fix-failing-tests-in-test_api.py.patch
+Patch10006:     0006-shell-assume-shell-plugins-are-in-etc.patch
+
+BuildArch:      noarch
+
+BuildRequires:  bash-completion
```

Comment:
>bash-completion [](http://example.com/codeflow?start=16&length=15)

This should technically be the devel subpackage of bash-completion. In practice we made bash-completion-devel a runtime requirement of bash-completion because many specs expected the pkgconfig file to be in the base package... but bash-completion-devel would be the most future-proof here